schema/deploy/shipping/views.sql

```diff
@@ -558,3 +560,35 @@ comment on view shipping.observation_with_presence_absence_result_v2 is
   'Joined view of shipping.incidence_model_observation_v3 and shipping.presence_absence_result_v1';
 
 commit;
+
+create or replace view shipping.metadata_for_augur_build_v3 as
+
+    select sample as strain,
+            coalesce(
+              cast(encountered as date),
+              case
+                when date_or_null(sample.details->>'date') <= current_timestamp::date
+                  then date_or_null(sample.details->>'date')
+              end
+            ) as date,
+            'seattle' as region,
+            -- XXX TODO: Change to PUMA and neighborhoods
+            residence_census_tract as location,
+            'Seattle Flu Study' as authors,
+            age_range_coarse,
+            case
+                when age_range_coarse <@ '[0 mon, 18 years)'::intervalrange then 'child'
+                else 'adult'
+            end as age_category,
+            warehouse.site.details->>'category' as site_category,
+            residence_census_tract,
+            flu_shot,
+            sex
+
+      from warehouse.sample
+      left join warehouse.encounter using (encounter_id)
+      left join shipping.incidence_model_observation_v2 on encounter.identifier = incidence_model_observation_v2.encounter
```

Comment:
I just realized that these joins aren't quite right if an encounter has multiple samples: they'll produce the cross-product of the `warehouse.sample` rows and the `shipping.incidence_model_observation_v2`, each of which has one row per sample. The join on `shipping.incidence_model_observation_v2` should use `sample.identifier = incidence_model_observation_v2.sample`.

Reply:
Similarly, I think that it'd be good to filter out `warehouse.sample` rows without an `identifier`. These will be ones for which we only know the `collection_identifier` so far.